lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -177,7 +177,7 @@ def version(prog=None):
     result.append("  ansible python module location = %s" % ':'.join(ansible.__path__))
     result.append("  ansible collection location = %s" % ':'.join(C.COLLECTIONS_PATHS))
     result.append("  executable location = %s" % sys.argv[0])
-    result.append("  python version = %s" % ''.join(sys.version.splitlines()))
+    result.append("  python version = %s (%s)" % (''.join(sys.version.splitlines()), to_native(sys.executable)))
```

Comment:
Out of curiosity why `to_native` and not `to_text`?

Reply:
for 3.x python there is really no diff, but if being backported, you need to_native for py2 otherwise we would have to force the other strings to `u''`